contrib/hamilton/contrib/user/zilto/lancedb_vdb/README.md

```diff
@@ -1,14 +1,17 @@
 # Purpose of this module
 
-This module implements simple operations to interact with LanceDB.
+This module implements vector and full-text search using LanceDB.
 
 # Configuration Options
 This module doesn't receive any configuration.
 
-## Inputs:
+## Required Inputs:
```

Comment:
Why "required"?

Reply:
I thought that `Inputs` might imply that is an exhaustive list of all the inputs. Instead, this list includes that you will have to pass as input to use the module albeit not all concurrently

Reply:
Yeah I think `Inputs` is fine. No need to for that extra qualification. Hamilton should complain anyway if something isn't satisfied.